apis/r/DESCRIPTION

```diff
@@ -4,7 +4,7 @@ Title: TileDB SOMA
 Description: Interface for working with 'TileDB'-based Stack of Matrices,
     Annotated ('SOMA'): an open data model for representing annotated matrices,
     like those commonly used for single cell data analysis.
-Version: 1.4.4
```

Comment:
Appending a dev number to the previous released version (1.4.3) since we haven't released 1.4.4 yet.

Reply:
FWIW I 'fixed' this too by reverting main to 1.4.3 in one of the PRs from yesterday.